cmd/minikube/cmd/start.go

```diff
@@ -997,7 +997,9 @@ func getKubernetesVersion(old *config.ClusterConfig) string {
 
 	}
 	if defaultVersion.GT(nvs) {
-		out.T(out.ThumbsUp, "Kubernetes {{.new}} is now available. If you would like to upgrade, specify: --kubernetes-version={{.new}}", out.V{"new": defaultVersion})
+        out.T(out.New,
+              "Kubernetes {{.new}} is now available. If you would like to upgrade, specify: --kubernetes-version={{.new}}",
```

Comment:
why there is a new line after ,  could we keep it in same line?

Reply:
Ohhh my old habits from OpenStack.